src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -16,7 +16,7 @@ public abstract class BaseGetSetTimes<T> : FileSystemTest
         // AppContainer restricts access to DriveFormat (::GetVolumeInformation)
         private static string driveFormat = PlatformDetection.IsInAppContainer ? string.Empty : new DriveInfo(Path.GetTempPath()).DriveFormat;
 
-        protected static bool isHFS => driveFormat.Equals(HFS, StringComparison.InvariantCultureIgnoreCase);
+        protected static bool isHFS => driveFormat == null ? false : driveFormat.Equals(HFS, StringComparison.InvariantCultureIgnoreCase);
```

Comment:
I think this might more commonly be written `driveFormat!= null && driveFormat.Equals(HFS, StringComparison.InvariantCultureIgnoreCase);`